packages/quantic/docs/jsdoc-guidelines.md

```diff
@@ -0,0 +1,72 @@
+# JSDoc Guidelines for Quantic Components
+
+The Quantic library has [reference documentation](https://docs.coveo.com/en/quantic/latest/reference/) that is automatically generated from JSDoc comments in the component source code. The component comments are parsed by JSDoc and the resulting doclet data is parsed by our custom template script found in `quantic/docs/template/lwc-json/publish.js`. This document outlines the required attributes and format of these JSDOC comments.
+
+## Type Imports
```

Comment:
Just to make sure: it's normal that we don't leverage this in the documentation right now?

Reply:
Oh I hadn't noticed we didn't. I guess since there is no description of th event and when it is dispatched it's probably not important to include it yet... Frankly IDK.

Reply:
Yeah I don't really see why someone would need that now 🤔 
Even in the future, I don't see why someone would need that info?